crates/turbopack-tests/tests/execution.rs

```diff
@@ -263,16 +263,28 @@ async fn run_test(resource: String) -> Result<Vc<RunTestResult>> {
         ])),
         vec![],
         Completion::immutable(),
-        false,
+        should_debug("execution_test"),
     )
     .await?;
 
-    let SingleValue::Single(bytes) = res
+    let single = res
         .try_into_single()
         .await
-        .context("test node result did not emit anything")?
-    else {
-        panic!("Evaluation stream must yield SingleValue.");
+        .context("test node result did not emit anything")?;
+
+    let SingleValue::Single(bytes) = single else {
+        return Ok(RunTestResult {
```

Comment:
Should this be returning an error instead, since we can't actually test the output?

Reply:
The code calling this function checks if the results are empty, so it's not necessary.

We also can't collect issues if we return an error unfortunately.